src/test/resources/edu/cornell/kfs/tax/batch/output-definition-test/bad-file-static-field-with-derived-key.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<taxOutputDefinition fieldSeparator=",">
+    <section name="Example_Section">
+        <field name="recipient_id" length="25" type="STATIC" key="recipientId"/>
```

Comment:
In general, is the length field needed since we are generating a CSV that will have variable lengths?

Reply:
Even for tax files that don't enforce fixed field lengths, they usually still expect a maximum length for certain fields. The spec spreadsheet for Sprintax also contains some fields that expect specific lengths or maximum lengths (Chapter 3 Status Code, the various dollar amount fields, etc.).

Would it make more sense to rename this attribute as a max length (such as "maxlength" or "maxLength" or "max-length")? Or do you think the current "length" name is clear enough?

Reply:
Okay, thank you for the explanation, I'm good with leaving it "length"